frontends/ol-components/src/components/TitledCarousel/TitledCarousel.tsx

```diff
@@ -48,13 +48,52 @@ const ButtonsContainer = styled.div`
   align-items: center;
 `
 
+const CAROUSEL_SPACING = 24
+
+const StyledCarousel = styled(Carousel)({
+  paddingTop: theme.custom.shadowOverflowTop,
+  paddingBottom: theme.custom.shadowOverflowBottom,
+
+  /*
+    We want the carousel cards to be:
+      1. spaced
+      2. cast a shadow when hovered
+      3. left-aligned (left edge of left-most card aligned with rest of page content)
```

Comment:
very minor nit on what might be my writing 😛 

very minor nit on what might be my writing 😛 

```suggestion
    We want the carousel cards to:
      1. be spaced
      2. cast a shadow when hovered
      3. be left-aligned (left edge of left-most card aligned with rest of page content)
```